pod_test.go

```diff
@@ -54,6 +55,10 @@ func TestPodCollector(t *testing.T) {
 		# TYPE kube_pod_status_ready gauge
 		# HELP kube_pod_status_scheduled Describes the status of the scheduling process for the pod.
 		# TYPE kube_pod_status_scheduled gauge
+		# HELP kube_pod_container_requested_cpu_millicores The number of requested cpu millicores by a container.
+		# TYPE kube_pod_container_requested_cpu_millicores gauge
```

Comment:
Those should be base units, i.e. "cores", "bytes", "seconds", etc.
As the exposition is floating-point, no particular is better. But sticking with the simplest one everywhere, avoids having to account for it in queries.